src/app/app.component.ts

```diff
@@ -1,10 +1,81 @@
-import { Component } from '@angular/core';
-
+import { Component, OnInit  } from '@angular/core';
+import { discardPeriodicTasks } from '@angular/core/testing';
+import { Injectable, Pipe, PipeTransform } from '@angular/core';
+import{SearchFilterPipe} from './filter.pipe'
 @Component({
   selector: 'app-root',
+
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.css']
+  styleUrls: ['./app.component.css'],
+
 })
-export class AppComponent {
-  title = 'app';
+export class AppComponent implements OnInit{
+  tasks: Task[] = [];
+  newTask = "  ";
```

Comment:
W JS stosuje zawsze pojedyncze cudzysłowy: `'`. Tutaj wystarczyłoby w sumie gdybyś po prostu nadał typ `string`. Jeśli już przypisujesz taką wartość początkową to przypisuj ją pustą a nie już z jedną spacją w środku.

Reply:
https://github.com/jedi800/Checklistaa/pull/1/commits/ad734be2b539dabf26729a20f8759a4068d254f3